internal/store/postgres/model/resource.go

```diff
@@ -66,10 +66,9 @@ func (m *Resource) FromDomain(r *domain.Resource) error {
 
 	if r.ID != "" {
 		uuid, err := uuid.Parse(r.ID)
-		if err != nil {
-			return fmt.Errorf("parsing uuid: %w", err)
+		if err == nil {
+			m.ID = uuid
```

Comment:
We should not remove error handling of invalid uuid from here, unless there's a specific reason for this?